src/_blog/go-libp2p-2022.md

```diff
@@ -0,0 +1,118 @@
+
+Without further ado, let's take a look at notable accomplishments in the last year:
+
+## New Features ✨
+
+### Transport Protocols 🚚
+
+#### WebTransport 📡
+
+One of the most exciting developments of the year was the release of the WebTransport protocol in [v0.23.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.23.0). WebTransport enables browser-to-server connectivity in go-libp2p when paired with a peer running [js-libp2p-webtransport](https://www.npmjs.com/package/@libp2p/webtransport) in the browser.
+
+To learn more about this exciting feature, check out our blog post on [WebTransport in libp2p](https://blog.libp2p.io/2022-12-19-libp2p-webtransport/) and the [WebTransport documentation](https://docs.libp2p.io/concepts/transports/webtransport/).
+
+#### WebRTC (Browser to Server) 🛰️
+
+In addition to WebTransport, the go-libp2p team also began work on enabling the WebRTC transport, in partnership with Little Bear Labs. This new transport will allow for connectivity between go-libp2p server nodes and js-libp2p browser nodes using [js-libp2p-webrtc](https://github.com/libp2p/js-libp2p-webrtc). 
+
+While this feature is still under development and testing, you can check out a proof-of-concept demo of it in action on our [libp2p Day 2022 Recap blog post](https://blog.libp2p.io/2022-11-22-libp2p-day-2022-recap/#why-webrtc). 
+
+We expect to release this feature in Q1 2023.
+
+
+#### QUIC 🐰
+
+We also made developments to the existing QUIC implementation in go-libp2p. In go-libp2p, we maintained support for two versions of QUIC, [RFC 9000](https://datatracker.ietf.org/doc/) and the [draft-29 version](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport-29). In [v0.24.0](https://github.com/libp2p/go-libp2p/releases/tag/v0.24.0), go-libp2p changed two things. The first was to properly distinguish between these two QUIC versions (in their multiaddresses) and second was to change the default dial behavior. Previously, go-libp2p nodes would dial draft-29 by default but after v0.24.0, nodes prefer the new QUIC version. This was partly inspired by the [alpha release of the QUIC implementation in rust-libp2p](https://github.com/libp2p/rust-libp2p/releases/tag/v0.50.0).
+
+<!-- To learn more about how different versions of QUIC work please read: https://github.com/libp2p/docs/pull/238 -->
+<!-- above docs is a blocker for merge -->
+
+### Faster Handshakes 🤝
+
+In the second half of 2022, the go-libp2p team began working to decrease the TTFB (time to first byte) and increase the speed of connection establishment (for transports that don't have native stream multiplexing like TCP and WebSocket). The first part of this effort was the [Early Muxer Negotiation feature](https://github.com/libp2p/specs/blob/master/connections/inlined-muxer-negotiation.md).
```

Comment:
Should this link to the docs page instead?

Reply:
Now that it exists, yes will do